### PR TITLE
[DO NOT MERGE] Vision GA – vision_localize_objects & vision_localize_object_gcs

### PR DESCRIPTION
**Pull request by nirupa-kumar**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [x] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by nirupa-kumar** [comment]
@beccasaurus Please have a look

File: README.md [hunk 1]
```diff
@@ -9,60 +9,31 @@
 [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/googleapis/nodejs-vision?branch=master&svg=true)](https://ci.appveyor.com/project/googleapis/nodejs-vision)
```
Comment (beccasaurus) [inline README.md]: Where are these changes to the root README coming from?

I just checked out the `master` of `nodejs-vision` and ran `npm run generate-scaffolding` in the root & there were no updates to the root README

File: samples/detect.js [hunk 2]
```diff
@@ -831,6 +831,70 @@ function detectPdfText(bucketName, fileName) {
...
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision');
+
+  // Creates a client
+  const client = new vision.ImageAnnotatorClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const uri = `gs://bucket/bucketImage.png`;
```
Comment (beccasaurus) [inline samples/detect.js]: `const gcsUri`

File: samples/system-test/detect.test.js [hunk 3]
```diff
@@ -297,3 +298,27 @@ test(`should extract text from pdf file`, async t => {
...
+    cwd
+  );
+  t.true(
+    output.includes(`Name: Bird`) &&
+      output.includes(`Name: Duck`) &&
+      output.includes(`Name: Toy`)
+  );
+});
+
+test(`should detect 'Bird'in duck_and_truck.jpg in GCS bucket`, async t => {
```
Comment (beccasaurus) [inline samples/system-test/detect.test.js]: type `'Bird'in duck`

Maybe make this consistent with the existing tests

`should detect objects in a local file`

`should detect objects in a remote file`

Comment (beccasaurus) [reply samples/system-test/detect.test.js]: Ah, just noticed these weren't named by you!

Still: would be a good change to add the consistency :)

File: README.md (hunk 1 above)
Comment (nirupa-kumar) [reply README.md]: Sorry I'm not sure where that came from. Reverting in next commit.

Comment (nirupa-kumar) [reply README.md]: Fixed in latest commit.

File: samples/detect.js (hunk 2 above)
Comment (nirupa-kumar) [reply samples/detect.js]: Fixed in latest commit.

File: samples/system-test/detect.test.js (hunk 3 above)
Comment (nirupa-kumar) [reply samples/system-test/detect.test.js]: Fixed in latest commit.

**Comment by nirupa-kumar** [comment]
Fixed issues as per review comments.

**Comment by nirupa-kumar** [comment]
@beccasaurus Please let me know if there is anything else.

File: samples/system-test/detect.test.js [hunk 4]
```diff
@@ -297,3 +298,27 @@ test(`should extract text from pdf file`, async t => {
   );
   t.true(output.includes('pdf-ocr.pdf.json'));
 });
+
+test(`should detect objects in a local file`, async t => {
+  const output = await tools.runAsync(
+    `${cmd} localize-objects ${files[8]}`,
```
Comment (beccasaurus) [inline samples/system-test/detect.test.js]: Test fails – the command run is`$ node detect.js localize-objects [object Object]`

Change `files[8]` to `files[8].localPath`

**Pull request merged**